unittests/test_modules.py

```diff
@@ -25,7 +25,7 @@ def modules_system(request, monkeypatch):
     try:
         m = modules.ModulesSystem.create(*args)
     except ConfigError:
-        pytest.skip('{requst.param} not supported')
+        pytest.skip('{request.param} not supported')
```

Comment:
This should be an f-string:

```suggestion
        pytest.skip(f'{request.param} not supported')
```